Bugzilla/Search/Recent.pm

```diff
@@ -27,6 +27,9 @@ use constant LIST_ORDER => 'id DESC';
 use constant AUDIT_CREATES => 0;
 use constant AUDIT_UPDATES => 0;
 use constant AUDIT_REMOVES => 0;
+use constant USER_ID_FIELD => 'user_id';
+use constant NAME_FIELD    => 'name';
+use constant ID_FIELD      => 'id';
```

Comment:
I'm commenting here for the benefit of another reviewer -- it appears that ->id and ->user_id are different for this class.